arcgis-ios-sdk-samples/Display information/Show legend/MILLegendTableViewController.swift

```diff
@@ -28,7 +28,7 @@ class MILLegendTableViewController: UITableViewController {
         self.populateLegends(self.operationalLayers as AnyObject as! [AGSLayerContent])
     }
     
-    func populateLegends(layers:[AGSLayerContent]) {
+    func populateLegends(_ layers:[AGSLayerContent]) {
```

Comment:
Don't like the simplistic removal of _, the method doesn't flow well.
It should be something like `popuplateLegends(from layers:[])` or `populateLegends(fromLayers layers:[])`
See https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage